test/Microsoft.ML.Tests/TrainerEstimators/LbfgsTests.cs

```diff
@@ -84,21 +88,66 @@ public void TestLogisticRegressionWithStats()
             var transformer = pipe.Fit(dataView) as TransformerChain<BinaryPredictionTransformer<ParameterMixingCalibratedPredictor>>;
 
             var linearModel = transformer.LastTransformer.Model.SubPredictor as LinearBinaryModelParameters;
-            var stats = linearModel.Statistics;
-            LinearModelStatistics.TryGetBiasStatistics(stats, 2, out float stdError, out float zScore, out float pValue);
+            var stats = linearModel.Statistics as LinearModelParameterStatistics;
+
+            var biasStats = stats?.GetBiasStatistics();
+            Assert.NotNull(biasStats);
+
+            biasStats = stats.GetBiasStatisticsForValue(2);
+
+            Assert.NotNull(biasStats);
 
-            CompareNumbersWithTolerance(stdError, 0.250672936);
-            CompareNumbersWithTolerance(zScore, 7.97852373);
+            CompareNumbersWithTolerance(biasStats.StandardError, 0.250672936);
+            CompareNumbersWithTolerance(biasStats.ZScore, 7.97852373);
 
             var scoredData = transformer.Transform(dataView);
 
-            var coeffcients  = stats.GetCoefficientStatistics(linearModel, scoredData.Schema["Features"], 100);
+            var coefficients  = stats.GetCoefficientStatistics(scoredData.Schema["Features"], 100) ;
 
-            Assert.Equal(19, coeffcients.Length);
+            Assert.Equal(19, coefficients.Length);
 
-            foreach(var coefficient in coeffcients)
+            foreach(var coefficient in coefficients)
                 Assert.True(coefficient.StandardError < 1.0);
 
         }
+
+        [Fact]
+        public void TestLRWithStatsBackCompatibility()
```

Comment:
>TestLRWithStatsBackCompatibility [](start = 20, length = 32)

if this PR is acceptable, add a test about LR backwards compatibility for model with stats saved.